doc/system_health_monitoring/system-health-HLD.md

```diff
@@ -125,16 +125,16 @@ The configuration shall be:
 }
 ```
 
-### 1.5 system status LED color definition
+### 1.5 System status LED color definition
 
-default system status LED color definition is like
+System status LED is set based on the status of system. The system status is defined as following
+
+ | Status           | Led Color            | Description                                                               |
+ |:----------------:|:--------------------:|:-------------------------------------------------------------------------:|
+ | booting          | ${led_color.booting} | System up time is less that 5 mins and not all services/devices are ready |
+ | normal           | ${led_color.normal}  | All services/devices are in good state                                    |
+ | fault            | ${led_color.fault}   | Not all services/devices are in good state                                |
```

Comment:
Do get into fault state also when boot time is more than 5 min and not all is ok, right?

Reply:
yes